components/registration-framework/org.wso2.carbon.identity.user.registration.mgt/src/main/java/org/wso2/carbon/identity/user/registration/mgt/dao/SQLConstants.java

```diff
@@ -54,13 +81,22 @@ private SQLConstants() {
     public static final class SQLPlaceholders {
 
         public static final String REGISTRATION_FLOW = "REGISTRATION";
-        public static final String STEP_ID = "STEP_ID";
-        public static final String PAGE_CONTENT = "PAGE_CONTENT";
-        public static final String PAGE_TYPE = "PAGE_TYPE";
-        public static final String COORDINATE_X = "COORDINATE_X";
-        public static final String COORDINATE_Y = "COORDINATE_Y";
-        public static final String HEIGHT = "HEIGHT";
-        public static final String WIDTH = "WIDTH";
+        public static final String DB_SCHEMA_COLUMN_NAME_STEP_ID = "STEP_ID";
+        public static final String DB_SCHEMA_COLUMN_NAME_PAGE_CONTENT = "PAGE_CONTENT";
+        public static final String DB_SCHEMA_COLUMN_NAME_PAGE_TYPE = "PAGE_TYPE";
+        public static final String DB_SCHEMA_COLUMN_NAME_COORDINATE_X = "COORDINATE_X";
+        public static final String DB_SCHEMA_COLUMN_NAME_COORDINATE_Y = "COORDINATE_Y";
+        public static final String DB_SCHEMA_COLUMN_NAME_HEIGHT = "HEIGHT";
+        public static final String DB_SCHEMA_COLUMN_NAME_WIDTH = "WIDTH";
+        public static final String DB_SCHEMA_COLUMN_NAME_NODE_ID = "NODE_ID";
+        public static final String DB_SCHEMA_COLUMN_NAME_NODE_TYPE = "NODE_TYPE";
+        public static final String DB_SCHEMA_COLUMN_NAME_IS_FIRST_NODE = "IS_FIRST_NODE";
+        public static final String DB_SCHEMA_COLUMN_NAME_TRIGGERING_ELEMENT = "TRIGGERING_ELEMENT";
+        public static final String DB_SCHEMA_COLUMN_NAME_EXECUTOR_NAME = "EXECUTOR_NAME";
+        public static final String DB_SCHEMA_COLUMN_NAME_IDP_NAME = "IDP_NAME";
```

Comment:
Suggestion: Move all the column names to a class named `SQLColumn` or similar and remove the `DB_SCHEMA_COLUMN_NAME` from the constant name.

Reply:
We have used this pattern in other DAOs in the framework.